src/libraries/System.Private.CoreLib/src/System/Int16.cs

```diff
@@ -940,8 +1030,5 @@ public static bool TryCreate<TOther>(TOther value, out short result)
 
         /// <inheritdoc cref="IUnaryPlusOperators{TSelf, TResult}.op_UnaryPlus(TSelf)" />
         static short IUnaryPlusOperators<short, short>.operator +(short value) => (short)(+value);
-
-        // /// <inheritdoc cref="IUnaryPlusOperators{TSelf, TResult}.op_CheckedUnaryPlus(TSelf)" />
-        // static short IUnaryPlusOperators<short, short>.operator checked +(short value) => checked((short)(+value));
```

Comment:
I see that removing `op_CheckedUnaryPlus` was part of the proposal, but I don't remember talking about it.  Why are we removing it?

(My best guess is "unary+ already doesn't respect checked", since `checked((short)+(short.MinValue))` doesn't throw, but returns a negative number)

Reply:
And, I'm reminded that unary + isn't absolute value, it's... sort of nothing, except that byte/sbyte/short/ushort end up being int, and uint/long/ulong are just "return this".  Not sure it's even worth defining the unchecked version in generic math, since we don't allow it to widen the return type.

c'est la vie.

Reply:
Right, its a `nop` and the language isn't exposing support for `op_CheckedUnaryPlus`

Its being exposed because its a simple thing to support and is used in various contexts by users who want to align code, express explicit intent around sign, etc.

We'll end up making it a DIM (when the support goes in) so that most users don't even need to consider supporting it